docs/reference/ingest/ingest-node.asciidoc

```diff
@@ -1217,7 +1217,7 @@ expression that supports aliased expressions that can be reused.
 This tool is perfect for syslog logs, apache and other webserver logs, mysql logs, and in general, any log format
 that is generally written for humans and not computer consumption.
 This processor comes packaged with over
-https://github.com/elastic/elasticsearch/tree/master/modules/ingest-common/src/main/resources/patterns[120 reusable patterns].
+https://github.com/elastic/elasticsearch/blob/{branch}/libs/grok/src/main/resources/patterns/grok-patterns[120 reusable patterns].
```

Comment:
Sorry about nitpicking, but when I re-checked the target directory for the link, I saw that maybe the parent directory (https://github.com/elastic/elasticsearch/tree/master/libs/grok/src/main/resources/patterns) would be a better choice? I think the link originally pointed there (e.g. on 5.6 branch: https://github.com/elastic/elasticsearch/tree/5.6/modules/ingest-common/src/main/resources/patterns)
Maybe @mvg would also know if the number of patterns should be adapted while we are at it. Looks like way more thank 120 to me, are they all packaged with the grok processor or only a subset of them?